gson/README.md

```diff
@@ -0,0 +1,4 @@
+# gson
```

Comment:
I agree that the original `gson/README` file is not particularly helpful, since it is essentially a truncated form of the `README.md` file from the parent directory.